packages/plugin-presence/src/index.js

```diff
@@ -0,0 +1,17 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {registerPlugin} from '@ciscospark/spark-core';
+import Presence from './presence';
+import config from './config';
+
+import '@ciscospark/plugin-wdm';
```

Comment:
I don't think it matters, but i try to keep anonymous imports at the top of the file.